src/server/models/db/schema.prisma

```diff
@@ -120,16 +121,17 @@ model Feedback {
   @@index([type])
 }
 
+// actions that can be performed on anything that is not a ListItem. ListItems have their own events.
```

Comment:
By editing the prisma schema I'm curious as to how this will work in prod since we can't wipe the data and rebuild the tables 🤔

Reply:
we use `:deploy` on prod which is non destructive